README.md

```diff
@@ -106,6 +106,14 @@ $ docker run --env-file .env todo-app:test e2e_tests
 
 # Deploying the application
 
-The application is deployed with azure and is accessible at https://charlie-devops-to-do.azurewebsites.net/
+The application is deployed with azure and is accessible at https://prod-charlie-devops-to-do.azurewebsites.net/
```

Comment:
Not sure if you changed your plan, but I was actually able to access it at:
http://charlie-devops-to-do-terraform.azurewebsites.net/